Formula/sile.rb

```diff
@@ -25,6 +24,7 @@ class Sile < Formula
   depends_on "luarocks" => :build
   depends_on "pkg-config" => :build
   depends_on "fontconfig"
+  depends_on "git"
```

Comment:
Why does it depend on git now?

Reply:
It actually has for a while, we just didn't document it or check at build time properly. SILE has an internal plugin system (package manager) that uses system calls to `git` to fetch plugins. You can use other functionality without it, but the package manager will die with an error.

This is mentioned in the release notes for v0.11.1.

Reply:
Maybe `uses_from_macos "git"` would be better then. Unless it needs a very specific version.

Reply:
Sure, if that's a reliable way to have `git` in the path at run time that sounds fine. I just made the switch.